golf2/src/main.rs

```diff
@@ -431,20 +431,22 @@ pub unsafe fn reset_handler() {
                                        VirtualMuxAlarm<'static, Timels>>>::new(flash));
 
     #[allow(unused)]
-    let nvcounter_test = static_init!(
+    /*let nvcounter_test = static_init!(
         nvcounter_test::NvCounterTest<'static, FlashCounter<'static,
             virtual_flash::FlashUser<'static>>>,
         nvcounter_test::NvCounterTest::new(nvcounter));
-    nvcounter.set_client(nvcounter_test);
+    nvcounter.set_client(nvcounter_test); */
 
     // dcrypto_test::run_dcrypto();
     //    rng_test::run_rng();
     //flash_test.run();
     //nvcounter_test.run();
 
     // Uncomment to initialize NvCounter
+    nvcounter.set_client(nvcounter_syscall);
```

Comment:
This is actually done on line 297, which is more consistent with the rest of this file. Commenting line 438 is what's necessary here (which you did as well).